app/packs/stylesheets/decidim/decidim_application.scss

```diff
@@ -6,10 +6,5 @@
 //
 // By default this is empty.
 
-@import "decidim/vizzs/areachart";
-@import "decidim/email";
-@import "custom-footer";
+@import "email/email-custom";
```

Comment:
@paulinebessoles we check if css is applied or not, with last commit we are duplicating the email-custom scss file in the pack